src/config.rs

```diff
@@ -510,26 +589,97 @@ impl Cfg {
         Ok(None)
     }
 
+    fn parse_override_file<S: AsRef<str>>(
+        contents: S,
+    ) -> (Option<OverrideFile>, Option<ErrorKind>) {
+        let contents = contents.as_ref();
+
+        // Avoid TOML parsing error for backwards compatibility with the legacy format
+        if contents.is_empty() {
```

Comment:
This could go in the 'in case of error' block I think, or does lines() not return an empty line for an empty file?

Reply:
You are right, in fact, I think I can remove the code completely as long as we ignore the toml error

Reply:
This is not necessary anymore with the new parsing strategy.